pages/favorite/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Favorite from '../../components/Favorite';
+import Layout from "/components/Layout";
+import { useState } from 'react';
+
+
+export default function FavoritePage(props) {
+
+    const [productData, setProductData] = useState([]);
+  return (
```

Comment:
Looks like prettier is not working on save; please check the settings in VSCode to make sure the code is formatted accordingly.